tslint.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./node_modules/@balena/lint/config/tslint-prettier.json",
   "rules": {
-    "no-shadowed-variable": false
+    "no-shadowed-variable": false,
+    "no-bitwise": false
```

Comment:
I'd rather keep this (even if it means disabling a single line somewhere), since bitwise operations are pretty difficult to read and reason about.

Reply:
Yeah - not even sure why this ended up in the tslint file as I had a single line disable comment in place anyway!

Reply:
...ahhhh - it's because I was using an inline `eslint-disable-next-line` comment rather than `tslint:disable-next-line`. Fixed now.